demo/app.js

```diff
@@ -65,33 +66,32 @@ export default function App() {
     />
   ));
 
+  const carouselProps = {
+    animation,
+    autoplay,
+    cellAlign,
+    cellSpacing,
+    enableKeyboardControls,
+    heightMode,
+    scrollMode,
+    slideIndex,
+    slideListMargin: 0,
+    slidesToScroll,
+    slidesToShow,
+    transitionMode,
+    withoutControls,
+    wrapAround,
+    zoomScale: Number(zoomScale || 0),
+    renderAnnounceSlideMessage: ({ currentSlide, slideCount }) =>
+      `Showing slide ${currentSlide + 1} of ${slideCount}`,
+    renderTopCenterControls: ({ currentSlide }) =>
+      renderTopControls(currentSlide)
+  };
```

Comment:
Extracted this into an object so we can reuse it to display a preview (Line 292)